### PR TITLE
Ad industry part deux: invasive targeting

### PR DESCRIPTION
**Pull request by davidfischer**

### CODE REVIEW

File: content/posts/2023-ad-industry-overview-part-ii.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+The same technique can be used to show ads to users who watch a specific video, launch a specific mobile app, search for specific terms, or even with a bit of work (Facebook and Google don't make this easy, other adtech companies aren't as picky) to target users who visit specific places in the real world.
+
+
+<div class="postimage text-center">
+  <img class="w-50 shadow-lg" src="{static}../images/posts/2023-ad-targeting-fbads-retargeting.png" alt="Retargeting options in Facebook Ads">
+  <p>Facebook Ads retargeting allows showing ads to visitors of specific web pages</p>
+</div>
+
+
+## Privacy concerns: crossing the line or good business
```
Comment (ericholscher) [inline content/posts/2023-ad-industry-overview-part-ii.md:51]:
```suggestion
## Privacy concerns: crossing the line or good business?
```

I think this is a question?

File: content/posts/2023-ad-industry-overview-part-ii.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+In our [previous post]({filename}../posts/2023-ad-industry-overview.md),
+we discussed ad exchanges, which facilitate the buying and selling of ad inventory in real-time.
+They present some especially acute privacy concerns.
+Firstly, they necessarily involve publishers sending personal data to the exchanges
+(usually obfuscated) for advertisers to bid on.
+Unlike the example of browsing Facebook or Google,
+it spreads copies of users' personal data to multiple companies,
+many of which don't have a direct relationship with the person being shown the ad.
+Lastly, it means that a data breach at any of the companies in the chain can expose personal data.
```
Comment (ericholscher) [inline content/posts/2023-ad-industry-overview-part-ii.md:77]: The Firstly/Lastly feels a bit odd here, given that there's only 2 items we introduce in that fashion. Might be fine, but read a bit weird to me.

File: content/posts/2023-ad-industry-overview-part-ii.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+
+## Does all this data result in better ads
+
+You would think that all this data would result in ad targeting better than you could possibly imagine.
+While some companies would lead you to believe that it's impossible to do advertising without tracking,
+that simply isn't true.
+Almost every single worldwide brand you can name was built with more traditional advertising
+before Internet surveillance advertising.
+In spite of all the tracking, users still consistently complain about seeing irrelevant ads.
```
Comment (ericholscher) [inline content/posts/2023-ad-industry-overview-part-ii.md:93]: Would it be worth talking about how often they also get data wrong, which leads to bad ads? Not sure if we need more content here tho.

File: content/posts/2023-ad-industry-overview-part-ii.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+[our vision]({filename}../pages/vision.md) for advertising is quite different.
+Instead of trying to build an ad network for everything, we focus exclusively on developers
+so we don't need to track users to show the right ad to the right user.
+Instead of letting any site onto our network with little or no oversight,
+we talk to and vet every single publisher on our network.
+Instead of letting any advertiser run anything in exchange for high ad bids,
+we make sure the products advertisers are promoting are real products relevant to our audience.
+No bogus sites, no scammy ads, and relevant products next to relevant content. That's it!
+
+While this focus on quality means we'll never grow to be as big as Google (or even 1/1,000th of Google),
```
Comment (ericholscher) [inline content/posts/2023-ad-industry-overview-part-ii.md:127]:
```suggestion
While this focus on quality means we'll never grow to be as big as Google,
```

Don't need to be too modest :D

**Review by ericholscher — APPROVED** [review]

File: content/posts/2023-ad-industry-overview-part-ii.md (hunk 3 above)
Comment (davidfischer) [reply content/posts/2023-ad-industry-overview-part-ii.md:93]: I think I'll skip that although it is relevant.

File: content/posts/2023-ad-industry-overview-part-ii.md (hunk 2 above)
Comment (davidfischer) [reply content/posts/2023-ad-industry-overview-part-ii.md]: I reworked it a bit.

**Pull request merged**